python/mxnet/libinfo.py

```diff
@@ -32,7 +32,11 @@ def find_lib_path():
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     api_path = os.path.join(curr_path, '../../lib/')
     cmake_build_path = os.path.join(curr_path, '../../build/')
-    dll_path = [curr_path, api_path, cmake_build_path]
+    clion_build_path_debug = os.path.join(curr_path, '../../cmake-build-debug/')
```

Comment:
Please don't do this.  Set LD_LIBRARY_PATH in CLion or set cmake's output directory in clion cmake options

Reply:
I would like to have this one. It causes no problem as only the first one is used, and makes it easier for newcomers and people using CLion to develop and contribute to our project. Could you reconsider your position on this particular issue?
Thanks a lot.

Reply:
This looks harmless but I don't like the idea of keep adding search paths either

Reply:
this is fine if you only ever build and run debug from Clion. many people, including me, often run Release or ReleaseWithDebInfo, so this would load Debug when what i really want is RelWithDebInfo, for instance. i already have this problem if anything is in “lib”. it’s not usually
obvious why it’s not working in that case.

Reply:
@piiswrong @cjolivier01 removed this for now since it's controversial. Can we add the fix now then?